.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+## Background
+<!-- What would a code reviewer, or a future dev, need to know about this PR in order to understand why this PR is necessary -->
+
+## Definition of Done
+
+- [ ] _Replace this line with individual tasks unique to your PR_
+- [ ] Appropriate test coverage & logging
+- [ ] Swagger docs have been updated, if applicable
```

Comment:
Should the README.md be included in the list of documentation to ensure gets updated if necessary?

Reply:
I am torn on that one.  I think it is a good reminder, and very low overhead, as it can just be tucked into this existing line.

On the other hand, IMHO, PR's that would be relevant to updating the `README` are too uncommon to justify being part of this "top 10 list", if you will.  Speaking only from my experience, I have shipped very few PR's, if any at all, that would involve updating our `README`.

But what do others think?

Reply:
I like the idea, but I think we should keep the list as minimal as possible for this iteration. 

I'd like to see more focus on swagger documentation (including overviews, definitions, feature descriptions, etc). Since the README is currently covering only installation/configuration and developer workflow, I think we're good to not call it out for now.